src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -189,4 +190,26 @@ public function deleteAction(Request $request, Post $post)
 
         return $this->redirectToRoute('admin_post_index');
     }
+
+    /**
+     * Converts a markdown string to HTML.
+     *
+     * @Route("/preview", name="preview")
```

Comment:
You can name this route `admin_post_content_preview` to be consistent with this action and others routes?